tslint.json

```diff
@@ -56,8 +56,6 @@
     "no-unbound-method": false,
     "no-unnecessary-callback-wrapper": false,
     "no-unsafe-any": false,
-    "no-unused-variable": false,
```

Comment:
Why removing those?

Reply:
cli this remove after update, this rule is deprecated: https://github.com/palantir/tslint/pull/3919